package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "jquery-focuspoint",
+  "version": "1.1.3",
+  "description": "jQuery plugin for 'responsive cropping'. Dynamically crop images to fill available space without cutting out the image's subject. Great for full-screen images.",
+  "main": "js/jquery.focuspoint.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jonom/jquery-focuspoint.git"
+  },
+  "keywords": [
+    "\"crop\""
+  ],
+  "author": "\"cropping\",",
```

Comment:
This part looks wrong... 'cropping' meant to be in keywords section? Author should be Jonathon Menz http://jonathonmenz.com (no email) Here is how you markup an author: https://docs.npmjs.com/files/package.json#people-fields-author-contributors